Headline feature: Module 1 skeleton mobile app

gabrielyea opened the pull request:
# Pull request for mobile app
- added style.css 
- added index.html file
- no linter errors
- added headline feature
- added toolbar mockup
- added all project images to the assets folder

gabrielyea committed:
add menu img in assets folder

gabrielyea committed:
add nav bar to main page

gabrielyea committed:
add style to navbar elements

gabrielyea committed:
add dependencies for linters

gabrielyea committed:
add icon images to assets folder

gabrielyea committed:
add content, information to the headline

gabrielyea committed:
add style to headline elements

gabrielyea committed:
Update README.md

Added description information

gabrielyea committed:
Add app screenshot

gabrielyea committed:
add screenshot and demo link

MahmoudBakr23 commented on README.md:
```diff
@@ -3,17 +3,18 @@
+> The mobile page should be a clone of [this template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)
 
-<!--![screenshot](./app_screenshot.png)-->
+Here you can see a picture ---> [screenshot](./app_screenshot.png)
```
- [x] Please fix this screenshot here. You can always take a look back at the README [template](https://github.com/microverseinc/readme-template) to remind yourself
- [x] Also, kindly fix the broken MIT link. I couldn't comment around the line downside because it's not from the changed files.
Please keep in mind that such a very small mistake could leave a bad impression on future reviewers.

MahmoudBakr23 commented on README.md:
```diff
@@ -3,17 +3,18 @@
-<!--## Live Demo-->
-
+## Live Demo
+https://gabrielyea.github.io/module1-portfolio-setup/
```
- [x] **OPTIONAL**
It's more clean and good-looking if you use the markup language to shorten the links.
Like so!
[Live Demo Link](https://gabrielyea.github.io/module1-portfolio-setup/)

MahmoudBakr23 commented on index.html:
```diff
@@ -0,0 +1,48 @@
+</head>
+
+<body>
+  <div class="main-container">
```
- [x] Great job with the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp), just another little thing. It'd be nice if you replaced the `<div>` tag with a `<header>` tag right? What do you think!

MahmoudBakr23 commented on index.html:
```diff
@@ -0,0 +1,48 @@
+  <div class="main-container">
+    <nav>
+      <div class="logo">
+        <span>Logo</span>
```
- [x] Please keep in mind that the logo should be a link (`<a>` tag) to the home page of the website for easier navigation.

MahmoudBakr23 commented on index.html:
```diff
@@ -0,0 +1,48 @@
+    </nav>
+    <div class="headline">
+      <div>
+        <h1>Hey There.
+          I’m Alex</h1>
+
+        <h2>I’am a Software Developer</h2>
+
```
- [x] Kindly delete any blank line or indentation.
Kindly remember that there should be no blank space without a good reason for it such as separating a logical code from another or two sections of code apart.

MahmoudBakr23 commented on index.html:
```diff
@@ -0,0 +1,48 @@
+          what you see and have a project you need coded,
+          don’t hestiate to contact me.</p>
+      </div>
+      <div class="contact-info">
+        <a href="#"> <img src="assets/Icon-GitHub.png" alt="github logo"></a>
+        <a href="#"> <img src="assets/Icon-Linkedin.png" alt="linkedin logo"></a>
+        <a href="#"> <img src="assets/Icon-Angelist.png" alt="angelist logo"></a>
+        <a href="#"> <img src="assets/Icon-Medium.png" alt="medium logo"></a>
+        <a href="#"> <img src="assets/Icon-Twitter.png" alt="twitter logo"></a>
+      </div>
```
- [x] Kindly notice that this social-links list should be wrapped inside a list (`<ul>` tag), (`<li>` tag), and finally the links (`<a>` tags).

_Please look up the comment about Semantic HTML._

MahmoudBakr23 reviewed (changes requested):
Hello @gabrielyea 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is well-configured 💯 
- Descriptive PR headline, summary, and commit messages 💯 
- Excellent follow to the given design 💯 
- Extra credit for the Live Demo Link 💯 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄


 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the **OPTIONAL** comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gabrielyea committed:
Create MIT.md

gabrielyea committed:
fixed typo in screenshot and broken links

gabrielyea committed:
change live demo link to markup

gabrielyea committed:
fix semantic error main div should be header

gabrielyea committed:
put contact info icons inside a list

gabrielyea committed:
add inline-flex display so list shows up correctly

gabrielyea committed:
made logo a link in index.html

gabrielyea commented:
# Fixed all observations and corrections
- deleted unnecessary empty lines in HTML file
- fixed semantic error
- fixed broken links and typos
- all social media links are on a list
- logo is now a link

> Thanks for the amazing feedback!!

mricanho reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @gabrielyea! 🔥 

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

![surprised](https://user-images.githubusercontent.com/60631456/120575207-39876100-c3e6-11eb-97ec-594284e88b46.gif)


Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
